### PR TITLE
Add checkboxes to select countries

### PR DESCRIPTION
**Pull request by limli**
Screenshot 2020-06-05 at 9 43 08 AM

The checkboxes don't do anything yet, just shows if it is selected

### CODE REVIEW

**Comment by limli** [comment]
> Do you plan on populating a large list of countries? Else, it may be better to use `<select>`?
> 
> https://www.w3schools.com/tags/att_select_multiple.asp#:~:text=When%20present%2C%20it%20specifies%20that,button%20to%20select%20multiple%20options

I'm not sure how many countries I intend to add. I just felt that the select multiple ui looks a bit ugly, because I don't like holding the ctrl key to select multiple things. Maybe a scrollable list of checkboxes might be better?
Eg:
Screenshot 2020-06-08 at 10 30 14 AM

**Comment by SuyashLakhotia** [comment]
Either scrollable list of checkboxes or `select` tag is fine.

**Comment by limli** [comment]
Screenshot 2020-06-08 at 4 39 52 PM

Changed to `<ul>` of checkboxes.

File: portfolio/src/main/webapp/chart-script.js [hunk 1]
```diff
@@ -20,6 +23,33 @@ function init() {
...
 }
 
+/**
+ * Initializes country selector
+ */
+function initSelector() {
+  const countrySelector = document.getElementById('country-selector');
+
+  const checkboxes = document.createElement('ul');
+  for (let i = 0; i < checkboxValues.length; i++) {
```
Comment (SuyashLakhotia) [inline portfolio/src/main/webapp/chart-script.js:33]: `for (const value of checkboxValues)` is cleaner if you don't need to refer to the index

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

**Pull request merged**